CMakeLists.txt

```diff
@@ -283,6 +284,34 @@ endif()
 
 # TODO: Check if ADIOS2 is parallel when openPMD_HAVE_MPI is ON
 
+# External library: libcxxwrap-julia
+if(openPMD_USE_JULIA STREQUAL AUTO)
+    find_package(JlCxx 0.8.3)
+    if(JlCxx_FOUND)
+        set(openPMD_HAVE_JlCxx TRUE)
+    else()
+        set(openPMD_HAVE_JlCxx FALSE)
+    endif()
+    if(JULIA_FOUND)
+        set(openPMD_HAVE_JULIA TRUE)
+    else()
+        set(openPMD_HAVE_JULIA FALSE)
+    endif()
+elseif(openPMD_USE_JULIA)
+    find_package(JlCxx 0.8.3 REQUIRED)
+    set(openPMD_HAVE_JULIA TRUE)
+    set(openPMD_HAVE_JlCxx TRUE)
+else()
+    set(openPMD_HAVE_JULIA FALSE)
+    set(openPMD_HAVE_JlCxx FALSE)
+endif()
+if(openPMD_HAVE_JlCxx)
+    get_target_property(JlCxx_location JlCxx::cxxwrap_julia LOCATION)
+    get_filename_component(JlCxx_location ${JlCxx_location} DIRECTORY)
+    set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib;${JlCxx_location}")
```

Comment:
Memo to myself: generalize.

- Control RPATH install (default: on) via option.
- Add `JlCxx_location` only to `openPMD_jl` target

Reply:
Prepared in #1105

Reply:
I have pushed a commit adapting to this change d427b1917ab8f40b14b2f21ac015e21ec018097a
Would probably be good if you look at that again @ax3l @eschnett